### ISSUE TITLE
Notebook screen remains empty on Windows

### ISSUE THREAD

**Issue by rikhuijzer** [JuliaLang/IJulia.jl#831]
I've installed IJulia via the package manager on Windows 10.
   
    versioninfo()    
    Julia Version 1.1.0
    Commit 80516ca202 (2019-01-21 21:24 UTC)
    Platform Info:
       OS: Windows (x86_64-w64-mingw32)
       CPU: Intel(R) Core(TM) i7-5930K CPU @ 3.50GHz
       WORD_SIZE: 64
       LIBM: libopenlibm
       LLVM: libLLVM-6.0.1 (ORCJIT, haswell)

When running `notebook()` in Julia the browser opens a blank screen. I've been going through the following steps.
1. Reinstall IJulia package. No change. Tested on both Firefox and Chrome.
2. Reset complete browser history on Firefox and Chrome. No change.
3. Installed Anaconda, and by that Jupyter Notebook. Can now run Julia notebooks succesfully via starting `Anaconda Prompt` and running `jupyter notebook`. 
4. Reinstalled and rebuild IJulia after running `ENV["JUPYTER"]="jupyter"` inside Julia. No change.

TL;DR: `using IJulia; notebook()` does not work for me. Workaround is to run Jupyter Notebook via Anaconda and hope that Jupyter will detect the Julia installation.

**Comment by stevengj** [comment]
What does `IJulia.find_jupyter_subcommand("notebook")` return?

**Comment by rikhuijzer** [comment]
I've removed some information and tried to include important parts. Let me know if you need more.

     setenv(`'C:\Users\Rik\.julia\conda\3\Scripts\jupyter.exe' notebook`,["PATH=C:\\Users\\Rik\\.julia\\conda\\3\\Scripts;C:\\Users\\Rik\\bin; ... ;C:\\Users\\Rik\\AppData\\Local\\Julia-1.1.0\\, ..., "EXEPATH=C:\\Program Files\\Git"])

**Comment by stevengj** [comment]
What happens if you run `C:\Users\Rik\.julia\conda\3\Scripts\jupyter.exe notebook` in the console?

**Comment by rikhuijzer** [comment]
I get the following error

    Error executing Jupyter command 'notebook': [Errno 'jupyter-notebook' not found] 2

**Comment by stevengj** [comment]
You may need to add `C:\Users\Rik\.julia\conda\3\Scripts` to your PATH in the console

**Comment by rikhuijzer** [comment]
Inside Julia I've executed `setenv(notebook(), "C:\\Users\\Rik\\.julia\\conda\\3\\Scripts")` and got output similar to the output above. This time it start with 

      running setenv(`'C:\Users\Rik\.julia\conda\3\Scripts\jupyter.exe' notebook`,["PATH=C:\\Users\\Rik\\.julia\\conda\\3\\Scripts;C:\\Users\\Rik\\bin;

The browser still shows `Home` as title with a Jupyter icon after being redirected to `http://localhost:8888/tree`

**Comment by stevengj** [comment]
Do you still have your Anaconda Jupyter running?  They might be conflicting…

(When IJulia is built it installs the Julia kernel spec in a standard global Jupyter configuration directory, so it is safe to assume that your Anaconda Jupyter will see Julia.)

**Comment by rikhuijzer** [comment]
Nope, that does not seem to be the problem. What does change is the port number. If Jupyter is running on 8889 then IJulia goes to 8890 otherwise IJulia goes to 8888.

I guess this is a very specific problem and would not mind if it is not solved. For me running Jupyter directly is even more convenient than starting julia and typing `using IJulia; notebook()`. So lets close the issue for now? Let me know if you want more info to solve it.

**Comment by stevengj** [comment]
The only other thing I can think of is to try
```jl
using Conda   # requires "add Conda" at the package prompt first
Conda.update()
```
to see if your Conda installation is out of date for some reason.

But if no one else can reproduce the problem, it will be hard to resolve.

**Comment by rikhuijzer** [comment]
Nice one! That one solved it. Thanks
